test/JDBC/expected/test_windows_login-vu-verify.out

```diff
@@ -0,0 +1,13 @@
+-- psql
+select rolname, orig_loginname, type, default_database_name, default_language_name from sys.babelfish_authid_login_ext;
```

Comment:
We should put a filter around babelfish_authid_login_ext. 

Also, add test case to check pg_authid, sys.server_principals, etc.

Reply:
Done!